crates/ironrdp-web/src/session.rs

```diff
@@ -297,11 +301,11 @@ impl SessionBuilder {
         loop {
             match ws.state() {
                 websocket::State::Closing | websocket::State::Closed => {
-                    return Err(IronRdpError::from(anyhow::anyhow!(
+                    return Err(RemoteDesktopError::from(anyhow::anyhow!(
                         "Failed to connect to {proxy_address} (WebSocket is `{:?}`)",
```

Comment:
Not something you added, but this should be:
```suggestion
                        "failed to connect to {proxy_address} (WebSocket is `{:?}`)",
```

Reply:
Done